packages/core/test/hotkeys/hotkeysTests.tsx

```diff
@@ -53,6 +53,21 @@ describe("Hotkeys", () => {
             }
         }
 
+        function assertInputAllowsKeys(type: string, allowsKeys: boolean) {
```

Comment:
I haven't looked at these test files recently, but I usually like to put helper functions last in the file / namespace scope / test suite. IMO this promotes file readability by letting you see the important things first -- the higher-level code that references these helper functions.